src/list-data/entity-type-data.ts

```diff
@@ -296,14 +299,8 @@ export const EntityTypesXproData: EntityI[] = [
       'Reinstate an extraprovincially registered company that is no longer active with BC Registries.'
     ],
     amlBlurbs: [
-      [
-        `Register an amalgamation that occurred in another jurisdiction where at least one of the companies is
-           extraprovincially registered in BC.`
-      ],
-      [
-        `Register an amalgamation or merger that occurred in another jurisdiction where at least one of the companies
-          is extraprovincially registered in BC.`
-      ]
+      `Register an amalgamation or merger that occurred in another jurisdiction where at least one of the companies
+        is extraprovincially registered in BC.`
```

Comment:
We can't distinguish between a Canadian or International amalgamation, so we don't need this array and I combined the verbiage.